glue/config.py

```diff
@@ -655,6 +654,40 @@ def default_members(self):
         return list(__FITTERS__)
 
 
+class KeyboardShortcut(DictRegistry):
+    """
+    Stores keyboard shortcuts.
+    The members property is a dictionary within a dictionary of keyboard
+    shortcuts, which is represented as (viewer,(keybind,function)). The
+    ``function`` should take one item, which is a reference to the session.
+    """
+
+    def add(self, valid_viewers, keybind, function):
+        """
+        Add a new keyboard shortcut
+        :param valid_viewers: list of viewers where event can be fired
+        :type label: list
+        :param keybind: type of key event
+        :type Qt.Key
+        :param function: function
+        :type function: function()
```

Comment:
Can you use the Numpydoc format for the docstring? (I'm trying to transition away from the sphinx format)